### DISCUSSION TITLE
Map-Midpoint

### DISCUSSION THREAD

**Discussion by ThomasMeister123** [osmandapp/OsmAnd#13983] [Ideas]
Hello,

I'm the only one who thinks that having the current position in the center of the map is pointless? Definitely in the option "orientation in the direction of travel".

The bottom half of the screen is completely pointless. What do I care about the route that I have already driven.

The current position should be in the middle at the BOTTOM

![x](https://user-images.githubusercontent.com/62500258/160246984-5165433d-a186-44f5-9f68-fa70aaba0056.png)

**Comment by pebogufi** [comment]
This works only if map orientation is driving direction. 
Your example shows compass, where it does not work.

**Reply by ThomasMeister123** [reply]
Shame on me. My fault :(

**Reply by Lee-Carre** [reply]
@ThomasMeister123

> works only if map orientation is driving direction.

For good reason. I'm often on foot (e.g. doing surveying), and it's frequently useful to know what's behind / around me, in any direction. Such as when needing a break, I want to find the nearest bench, regardless of direction. Backtracking is OK.

Also, sometimes I'm not using OsmAnd for _navigation_, but for general reference / context. So, I have no particular destination or route (imagine being out for a leisurely stroll, on a sunny afternoon), thus, any focus on only what's ‘ahead’ of me makes little sense. I'll randomly choose different lanes to wander down.

Giving priority to what's ahead **when in a vehicle** does make a lot of sense, but not necessarily in other contexts.

I feel that it comes down to how easily turns can be made.
* trains: only as the signalmen direct / allow; the route is predetermined
* aircraft: possible, but unlikely (for a given route), and unlikely to be >90° (except near airports)
* road vehicle (cars, bikes, trucks): can only turn at specific places, else have to find the likes of a carpark in which to turn around
* skiing: other than at ski lifts, will be heading downhill
* boats: on open water, can mostly turn freely
* on foot: pretty much at will, allowing for safety considerations; can turn on the spot and easily backtrack — on foot is notable in that distance is the dominant factor, due to low speed; the shortest route will be the quickest, and the nearest PoI (regardless of direction) will be quickest to reach

Maybe the best analogy is between a winged aircraft and a helicopter. The former must perform turns gradually, since it must maintain airspeed to remain aloft.
A helicopter can turn while otherwise stationary.
So, one may want the position marker centred for helicopter, even if not for winged aircraft.

**Comment by ThomasMeister123** [comment]
But another question: sometimes the symbol for orientation is NOT visible (compass, N,...) so I cant switch. I have to stop Navigation, change map and then the symbol is visible

**Reply by tommycrock** [reply]
Give the map a twist and I think it shows up again

**Reply by Lee-Carre** [reply]
> sometimes the symbol for orientation is NOT visible (compass, N,...) so I cant switch.

This is configurable in the settings for the screen (in OsmAnd, not your device OS); various UI widgets are optional (per-profile), including a button for changing orientation-mode.

**Reply by tommycrock** [reply]
I'd not found that before. Configure profile, Configure screen: Left panel, compass

**Comment by bschelm** [comment]
When using navigation, my map is always pointing north (north is top of screen), not in the direction of travel. Since a while (one or two updates ago), the midpoint moved from the middle of the screen to the bottom of the screen. That is very annoying because that makes it impossible to have the map pointing north. If I drive south, there are only very few pixels showing the way. this means, navigation is only useful when pointing the map in the direction of travel. It would make sense, that when pointing the map north, the midpoint (current location) would move to the middle of the screen.
Regards. Bernd

**Reply by tommycrock** [reply]
You need to set display position to centre for the relevant profile:
![Screenshot_20221209-223048](https://user-images.githubusercontent.com/5749884/217959938-7e98f17d-9b86-4df4-a8b6-f016373e3525.jpg)
![Screenshot_20221209-223106](https://user-images.githubusercontent.com/5749884/217960049-276e9616-874b-4a7e-8355-5cc35b6c62f5.jpg)
![Screenshot_20221209-223127](https://user-images.githubusercontent.com/5749884/217960053-99d7daf6-4ee5-4c44-94f1-2e100bae4a55.jpg)
![Screenshot_20221209-223143](https://user-images.githubusercontent.com/5749884/217960058-da9dfd91-17e5-4366-8957-c5f9ac3d5abb.jpg)

**Reply by bschelm** [reply]
Many many many thanks tommycrock.
I've been looking up and down for this option but were never able to find it until now.
